openshift/message-board/account-micro/pom.xml

```diff
@@ -0,0 +1,86 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.hibernate.demos.messageboard</groupId>
+    <artifactId>account-micro</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
+    <packaging>jar</packaging>
+
+    <name>Account MicroService - Thorntail v4</name>
+    <description>Preview of Account MicroService - using Thorntail v4</description>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+
+        <maven.compiler.target>1.8</maven.compiler.target>
+        <maven.compiler.source>1.8</maven.compiler.source>
+
+        <!-- TODO: looking forward to 4.0.0.Final -->
+        <thorntail.version>4.0.0-SNAPSHOT</thorntail.version>
```

Comment:
You used the `version.<project>` convention elsewhere but not here. We should try to be consistent across the whole project.